tofhir-server/src/main/scala/io/tofhir/server/endpoint/SchemaDefinitionEndpoint.scala

```diff
@@ -245,12 +251,14 @@ object SchemaDefinitionEndpoint {
   val SEGMENT_REDCAP = "redcap"
   val SEGMENT_IMPORT = "import"
   val SEGMENT_IMPORT_ZIP = "import-zip"
+  val QUERY_PARAM_TYPE = "type"
+  val QUERY_PARAM_URL = "url"
 }
 
 /**
  * The schema formats available for POST and GET schema methods
  */
-object SchemaFormats{
+object SchemaFormats {
   val STRUCTURE_DEFINITION = "StructureDefinition"
```

Comment:
Could you provide additional details in the comment to clarify the differences between SimpleStructureDefinition and StructureDefinition, particularly in relation to the SchemaDefinitionEndpoint?